docs/application/toc_all.md

```diff
@@ -78,6 +78,7 @@
 #### [Package Manager](/application/dotnet/guides/app-management/package-manager.md)
 #### [Widget Control](/application/dotnet/guides/app-management/widget-control.md)
 #### [Widget Viewer](/application/dotnet/guides/app-management/widget-viewer.md)
+#### [Cion](/application/dotnet/guides/app-management/cion.md)
```

Comment:
Line 81: I think Cion should be placed between [Application Preference] and [Component Manager] to maintain alphabetical order.
@mijin85cho please confirm this...

Reply:
it's moved